### ISSUE TITLE
model: pytorch: Add entrypoint style loading for optimizers and learning rate scheduler functions

### ISSUE THREAD

**Issue by sakshamarora1** [intel/dffml#840]
Currently only the torch.nn module has annotations.
Add PyTorchOptimizer and PyTorchScheduler after the next torch release as they are currently adding type annotations for every module in torch.
